easy-ui-react/src/FocusedProductLayout/FocusedProductLayout.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { ReactNode } from "react";
+import { MenuOverlayProps } from "../Menu/MenuOverlay";
+import { classNames, variationName } from "../utilities/css";
+import { Content } from "./Content";
+import { HeaderAtTopOfPage } from "./HeaderAtTopOfPage";
+import { HeaderInContentArea } from "./HeaderInContentArea";
+import { SidePanel } from "./SidePanel";
+import { WizardContent } from "./WizardContent";
+
+import styles from "./FocusedProductLayout.module.scss";
+
+export type HeaderProps = {
+  /**
+   * List of help menu items to render. Should be an array of `<Menu.Item />`s.
+   */
+  helpMenuItems: MenuOverlayProps<object>["children"];
+
+  /**
+   * Handler that is called when a help menu item is selected.
+   */
+  onHelpMenuAction?: MenuOverlayProps<object>["onAction"];
+
+  /**
+   * Back arrow to render in the header. The back functionality is completely
+   * up to the consumer.
+   *
+   * @example
+   * ```tsx
+   * <FocusedProductLayout
+   *   renderBackArrow={(props) => <a href="/account/settings" {...props} />}
+   * />
+   * ```
+   *
+   * @example
+   * ```tsx
+   * import Link from "next/link";
+   *
+   * <FocusedProductLayout
+   *   renderBackArrow={(props) => <Link href="/account/settings" {...props} />}
+   * />
+   * ```
+   *
+   * @example
+   * ```tsx
+   * <FocusedProductLayout
+   *   renderBackArrow={(props) => <button onClick={() => {}} {...props} />}
+   * />
+   * ```
+   */
+  renderBackArrow: (props: { children: ReactNode }) => ReactNode;
```

Comment:
**question** out of curiosity,  is the functionality left up to the consumer to allow for the possibility of using `next/link` and all the benefits we can get from that or is there another reason?

Reply:
yep that's precisely it. allowing the consumer to choose what happens when a user clicks "back". i left a short comment on the spec with my thoughts on the back arrow and logo render props: https://github.com/EasyPost/easy-ui/pull/703#discussion_r1360931283

let me know if you feel it should be different